### PR TITLE
proposal: add custom commit message example (s)

### PR DESCRIPTION
**Pull request by fuadop**
### Why:
This is related to https://github.com/dependabot/dependabot-core/issues/5192 .

cc: @jeffwidman

Closes [https://github.com/dependabot/dependabot-core/issues/5192 ]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 1]
```diff
@@ -280,6 +280,8 @@ updates:
       prefix-development: "pip dev"
       include: "scope"
 ```
+An example commit message based on your recent commit preferences would be `<prefix> (dev-deps): bump requests from 1.0.8 to 1.1.3`.
```
Comment (jeffwidman) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:283]: It looks like your example is based on your own personal settings, and not the settings in the above example config, which creates confusion.

IIUC, you've got `(dev-deps)` as the development prefix? It's unclear to me if your prefix includes the `(`,`)`, or if this is auto-added by Dependabot. 

Also, from the docs above it looks like the `prefix-development` will override the `prefix` entirely, not just be two prefixes in a row.

If the above config results in `pip dev: bump requests from 1.0.0 to 1.0.1` consider rewording to:
```suggestion
Example:
Using these preferences, bumping the `requests` library in the `pip` Development dependency group generates a commit message of:

   pip dev: bump requests from 1.0.0 to 1.0.1
   
```

But if I'm misunderstanding something with how this works, please clarify... or just take what I wrote above and tweak further.

Also, note to the Docs repo owners, my markdown formatting may be incorrect, or if you have a specific format for examples, then please fixup.

**Review by jeffwidman — COMMENTED** [review]
Thanks for putting up the PR! We always appreciate improvements to the docs.

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md (hunk 1 above)
Comment (fuadop) [reply content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:283]: Hey @jeffwidman , Yesss this is perfect !. Yea some of the prs have brackets.. for example: https://github.com/LearningTypeScript/projects/pull/31

**Comment by cmwilson21** [comment]
@fuadop Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 2]
```diff
@@ -280,6 +280,12 @@ updates:
       prefix-development: "pip dev"
       include: "scope"
 ```
+Example:
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:283]:
```suggestion
```

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 3]
```diff
@@ -280,6 +280,12 @@ updates:
...
+Using these preferences, bumping the `requests` library in the `pip` Development dependency group generates a commit message of:
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:284]:
```suggestion
If you use the same configuration as the example above, bumping the `requests` library in the `pip` development dependency group will generate a commit message of:
```

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 4]
```diff
@@ -280,6 +280,12 @@ updates:
...
+
+   pip dev: bump requests from 1.0.0 to 1.0.1
+   
+
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:288]: Remove new line
```suggestion
```

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 5]
```diff
@@ -280,6 +280,12 @@ updates:
       prefix-development: "pip dev"
       include: "scope"
 ```
+Example:
+Using these preferences, bumping the `requests` library in the `pip` Development dependency group generates a commit message of:
+
+   pip dev: bump requests from 1.0.0 to 1.0.1
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:286]:
```suggestion
   'pip dev: bump requests from 1.0.0 to 1.0.1'
```

**Review by mchammer01 — DISMISSED** [review]
@fuadop 👋🏻 - LGTM ✨ 
I've made a few minor comments, mainly to do with formatting and content models in our docs. Will apply the suggestions and then hopefully merge this PR.
Thank you for contributing to improve out docs :zap:

**Review by mchammer01 — APPROVED** [review]

**Comment by fuadop** [comment]
Thanks @mchammer01 @cmwilson21 @jeffwidman for your help on this 🎉❤️.

**Comment by tobiasdiez** [comment]
I'm confused by this remark in the docs and went to the code to understand the behavior. The commit message prefix is determined in (if I'm not mistaken):
https://github.com/dependabot/dependabot-core/blob/c00613fdce7f13eb832d5cd5ae9f0ac116edb50c/common/lib/dependabot/pull_request_creator/pr_name_prefixer.rb#L74-L82

If I'm reading this correctly, then with
```
commit-message:
      prefix: "pip prod"
      prefix-development: "pip dev"
      include: "scope"
```
the prefix would be:
- for production deps: `pip prod(deps)`
- for dev deps: `pip dev(deps-dev)`

Can someone please clarify this. Thanks!

**Pull request merged**